superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -36,6 +36,7 @@ interface ListViewResourceState<D extends object = any> {
   lastFetchDataConfig: FetchDataConfig | null;
   bulkSelectEnabled: boolean;
   lastFetched?: string;
+  defualtLoading?: boolean;
```

Comment:
```suggestion
  defaultLoading?: boolean;
```

Typo, plus this is probably not needed since the state itself doesn't really use `defaultLoading`.